### PR TITLE
feat: add animated Link

### PR DESCRIPTION
**Pull request by mhenriette**
## Describe your changes
Created a link component from ui folder and imported the diagonal arrow.  it has an  Animated arrow so that it moves a few pixels on the right and up when a user hovers over the link

## Issue ticket number and link
ID->#012
Link->[here](https://www.notion.so/apeunit/Link-component-91f3ef67c72d4694b37b4e44a33f4f50)

## Tasks completed

- [x] configure Line Heights in tailwind configuration file
- [x]  configure Letter Spacing in tailwind configuration file
- [x] create Link component call it Anchor
- [x] Animate Arrow to move top-right on hover

## Tasks not completed
none

### CODE REVIEW

File: components/ui/Anchor.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Image from 'next/image'
+import linkIcon from '/public/assets/icons/link-arrow.svg'
+import Link from 'next/link'
+
+export default function Anchor({href, children}) {
+  return (
+
```
Comment (hirwaaldo1) [inline components/ui/Anchor.jsx]: remove this space

File: tailwind.config.js [hunk 2]
```diff
@@ -21,10 +21,26 @@ module.exports = {
...
+      lineHeight:{
+        'tighter':72/64, //1.125
+        'tight':64/56, //1.14
+        'snug':42/36, //equal to 28:28/24, 1.1666
+        'normal': 34/28, //1.214
+        'relaxed':32/24, //1.333
+        'loose':28/20, //1.4
+      },
+      letterSpacing:{
+
```
Comment (hirwaaldo1) [inline tailwind.config.js]: here too remove the space

**Review by hirwaaldo1 — CHANGES_REQUESTED** [review]

File: tailwind.config.js [hunk 3]
```diff
@@ -21,10 +21,26 @@ module.exports = {
           100:'#878787' //this is  grey for the tags
         }
       },
+      lineHeight:{
+        'tighter':72/64, //1.125
+        'tight':64/56, //1.14
+        'snug':42/36, //equal to 28:28/24, 1.1666
```
Comment (mutesialine) [inline tailwind.config.js]: this comment is not understandable try to make it clear

**Review by hirwaaldo1 — APPROVED** [review]
it looks good

**Review by mutesialine — APPROVED** [review]
Well done its looks nice

**Pull request merged**